crypto/ct/ct_sct.c

```diff
@@ -223,6 +237,54 @@ int SCT_set1_signature(SCT *sct, const unsigned char *sig, size_t sig_len)
     return 1;
 }
 
+int SCT_set1_serialized_signature(SCT *sct, const unsigned char **in,
```

Comment:
You know what... now that I look at these... `set1` should probably take a `const unsigned char *in`, or will it ever need to read the signature whose complete length is not known in advance? And we shouldn't mix `int` and `size_t` returns in new APIs. (Same goes for `i2o_SCT`.)

But we really don't have time to think this through anymore, so let's revert to the `i2o`/`o2i` "convention" - at least then it's repeating old mistakes, not adding new ones...